tests/unit/Core/Framework/Validation/DataBag/DataBagTest.php

```diff
@@ -45,6 +45,53 @@ public function testConversion(): void
         static::assertEquals(['a' => 'b'], $bag->all('4'));
     }
 
+    public function testAddSetConvertsToDataBag(): void
+    {
+        $bag = new DataBag();
+
+        // test setting databag stays a databag, but converts to array
```

Comment:
why writing comments, if you could have test cases with those comments as method names instead? :wink:

Reply:
It was meant as an explanation what is about to be tested in comparison to other tests below